actions/snowflake-cortex/package.yaml

```diff
@@ -0,0 +1,41 @@
+# Required: A short name for the action package
+name: Snowflake Cortex
+
+# Required: A description of what's in the action package.
+description: Snowflake Cortex AI Actions
+
+# Package version number, recommend using semver.org
+version: 1.0.0
+
+spec-version: v2
+
+dependencies:
+  conda-forge:
+  - python=3.10.14
+  - uv=0.4.17
+  pypi:
+  - sema4ai-actions=1.3.5
+  - sema4ai-data=0.1.0
+  - snowflake-connector-python[pandas]=3.13.2
+  - pandas=2.2.3
+  - tabulate=0.9.0
+  - PyYAML=6.0.2
+  - openpyxl=3.2.0b1
+  - snowflake=1.0.4
```

Comment:
1.0.5 was released yesterday

Reply:
Don't want to risk it for this release as it works. Will update for next version.